subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaFirstUsePerformanceTest.groovy

```diff
@@ -26,7 +26,7 @@ class JavaFirstUsePerformanceTest extends AbstractCrossVersionPerformanceTest {
         given:
         runner.tasksToRun = ['tasks']
         runner.gradleOpts = ["-Xms${memory}", "-Xmx${memory}"]
-        runner.args = ['--recompile-scripts'] // This is an approximation of first use: we recompile the scripts
+        runner.args = ['--recompile-scripts', '--no-daemon'] // This is an approximation of first use: we recompile the scripts
```

Comment:
Should use runner.useDaemon=false, this is inspected by other parts of the infrastructure